### PR TITLE
DRILL-8251: Upgrade Hadoop 2 Due to CVE

### PR DESCRIPTION
**Pull request by pjfanning**
# [DRILL-8251](https://issues.apache.org/jira/browse/DRILL-8251): Upgrade hadoop 2 due to CVE

## Description

Relates to https://github.com/apache/drill/security/dependabot/21

## Documentation

Unknown

## Testing

CI build

### CODE REVIEW

**Review by cgivre — APPROVED** [review]
LGTM +1

@jnturton Is there anything special we need to do for this before we merge?

**Comment by jnturton** [comment]
> LGTM +1
> 
> @jnturton Is there anything special we need to do for this before we merge?

When we update our Hadoop 3 libs we must also update the winutils binaries we ship for Windows. But in the case of Hadoop 2 we do not support Windows, and I can't imagine anyone out there who wants that, so LGTM.

**Comment by cgivre** [comment]
@pjfanning It looks like you have two approvals.  If you want to mark this as ready for review, we can merge.   Thanks for this!

**Comment by pjfanning** [comment]
@cgivre thanks - marked as ready to review

**Pull request merged**
